hw/ip/tlul/generic_dv/xbar_sim_cfg.hjson

```diff
@@ -28,142 +28,12 @@
 
   // Import additional common sim cfg files.
   import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/data/common_sim_cfg.hjson"]
+                "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
+                "{proj_root}/hw/ip/tlul/data/xbar_tests.hjson"]
 
   // Add additional tops for simulation.
   sim_tops: ["-top {dut}_bind"]
 
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
-
```

Comment:
Was keeping this in the same file creating some kind of problem? I feel this could remain in the same file.

Reply:
this file isn't used for xbar_chip, as dut, excl file, common_sim_cfg, bind file are not needed. Only tests is needed. So I moved all the tests to another hjson file.

Reply:
Got it, thanks!